### DISCUSSION TITLE
Power Port

### DISCUSSION THREAD

**Discussion by JHarrison712** [netbox-community/netbox#12137] [Help Wanted!]
So I was adding my Power Port on my devices I have C13 but I don't have an option in Netbox for that type of Plug. Am I doing it wrong. Like a server has C13 on it > https://i.imgur.com/UNNGt7G.png

**Comment by markkuleinio** [comment]
That's not C13, it's C14 inlet: https://en.wikipedia.org/wiki/IEC_60320#C14

**Reply by JHarrison712** [reply]
Well I feel stupid, I do apologies for this question.
